control-plane-operator/controllers/hostedcontrolplane/kas/params.go

```diff
@@ -118,6 +121,12 @@ func NewKubeAPIServerParams(ctx context.Context, hcp *hyperv1.HostedControlPlane
 		params.Image = hcp.Spec.Configuration.Image
 		params.Scheduler = hcp.Spec.Configuration.Scheduler
 	}
+	if reqInflight := hcp.Annotations[hyperv1.KubeAPIServerMaximumRequestsInFlight]; reqInflight != "" {
+		params.MaxRequestsInflight = &reqInflight
```

Comment:
Do we want to do the defaulting at this level and pass `string` vs `*string`?  None of the other `KubeAPIServerConfigParams` fields are `*string`.

Reply:
updated